project_0a/project_0a_3.c

```diff
@@ -0,0 +1,30 @@
+/*
+Project 0a
+*/
+
+/*
+Exercise 3 - build a C to F converter. F = C × 1.8 + 32 that goes from 0 to 100 C, 1 by 1.
+*/
+
+#include <stdio.h>
+
+int main() {
+  // Convert C to F
+  // F = C × 1.8 + 32
+
+  float fahrenheit;
+  int celsius;
+
+  celsius = 0;
+
+  printf("\nCelsius to Fahrenheit convertion table\n");
+  
+  while (celsius <= 100) {
+    // Calculating
```

Comment:
This is obvious for anyone reading the code, doesn't need a comment. https://google.github.io/styleguide/cppguide.html#Implementation_Comments